app/src/assets/labware/getLabware.ts

```diff
@@ -7,50 +6,50 @@ import type {
 
 // require all definitions in the labware/definitions/1 directory
 // require.context is webpack-specific method
-const labwareSchemaV1DefsContext = (require: any).context(
+const labwareSchemaV1DefsContext = (require as any).context(
   '@opentrons/shared-data/labware/definitions/1',
   true, // traverse subdirectories
   /\.json$/, // import filter
   'sync' // load every definition into one synchronous chunk
 )
-let labwareSchemaV1Defs: $ReadOnlyArray<LabwareDefinition1> | null = null
-function getLegacyLabwareDefs(): $ReadOnlyArray<LabwareDefinition1> {
+let labwareSchemaV1Defs: Readonly<LabwareDefinition1[]> | null = null
+function getLegacyLabwareDefs(): Readonly<LabwareDefinition1[]> | null {
```

Comment:
I don't think `null` should be tacked on here; it can't return `null`

```suggestion
function getLegacyLabwareDefs(): Readonly<LabwareDefinition1[]> {
```